### PR TITLE
my-landing-page

### PR DESCRIPTION
**Pull request by Ntwali-Josue**
**What does this PR do**
This PR contains my landing page files(landing.html,landing.css)

**Description of Task to be completed?**
*Create header
*Create body
*Create footer

**What are the relevant Trello stories?**
*V7RHTbvY

### CODE REVIEW

File: landing page/landing.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header>
+        <h1>MY LOGO</h1>
+        <div class="nav-links">
+            <ul>
+                <li><a href="#">Home</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="/index.html">Contacts</a></li>
```
Comment (TuyizeeAnastase) [inline landing page/landing.html:17]: why this you named Contacts but is it your contact /index.html

File: landing page/landing.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="css/landing.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header>
+        <h1>MY LOGO</h1>
+        <div class="nav-links">
+            <ul>
+                <li><a href="#">Home</a></li>
```
Comment (TuyizeeAnastase) [inline landing page/landing.html:15]: can you use Home as your index.html? and contact for contact.html?? @Ntwali-Josue

File: landing page/landing.html (hunk 1 above)
Comment (Ntwali-Josue) [inline landing page/landing.html:17]: i'm trying to connect to my index.html but it's not seeing it!

Comment (TuyizeeAnastase) [reply landing page/landing.html:17]: what is your index.html is contact page or landing page?

File: landing page/landing.html (hunk 2 above)
Comment (Ntwali-Josue) [reply landing page/landing.html:15]: i've done that on contact but it's not working and let me try on home

File: landing page/landing.html (hunk 1 above)
Comment (TuyizeeAnastase) [reply landing page/landing.html:17]: use your homepage as index.html and contact as contact.html also check the location of your page

Comment (Ntwali-Josue) [reply landing page/landing.html:17]: by saying contact/index.html...do you mean the folder called contact?

Comment (Ntwali-Josue) [reply landing page/landing.html:17]: my index.html is the contact page

Comment (TuyizeeAnastase) [reply landing page/landing.html:17]: you can put there the link for your landingpage       <li><a href="#">Home</a></li>   like this   <li><a href="landingpage.html">Home</a></li> if you named as like landingpage.html

File: landing page/landing.html (hunk 2 above)
Comment (TuyizeeAnastase) [reply landing page/landing.html:15]: yes on contact page Home navigation put your landingpage.html  as href=""

File: landing page/landing.html (hunk 1 above)
Comment (Ntwali-Josue) [reply landing page/landing.html:17]: ``yeah let me work on it

File: index.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+function validation () {
+    var name = document.getElementById("name").value;
+    var email = document.getElementById("email").value;
+    var message = document.getElementById("msg").value;
+
+
+    error_message.style.padding = "10px";
+    var text;
+
+    if(name.length < 5){
```
Comment (TuyizeeAnastase) [inline index.js:10]: why are you limiting user name? can't be a name less than 5 character.

File: index.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        return false;
+    }
+
+    if(email.indexOf("@") == -1 || email.length < 6){
+        text = "Please Enter valid Email";
+        error_message.innerHTML = text;
+        return false;
+    }
+
+    if(message.length <= 5){
```
Comment (TuyizeeAnastase) [inline index.js:22]: why message can't be less than 5?

File: index.js (hunk 3 above)
Comment (TuyizeeAnastase) [reply index.js:10]: i think a name can be less than 5 character like Bob?

**Pull request merged**